cpp/arcticdb/version/version_store_api.cpp

```diff
@@ -1095,16 +1096,16 @@ std::pair<VersionedItem, py::object> PythonVersionStore::read_descriptor(
     } else {
         pyobj = pybind11::none();
     }
-    return std::make_pair(version, pyobj);
+    return std::make_pair(descriptor, pyobj);
 }
 
-std::vector<std::pair<VersionedItem, py::object>> PythonVersionStore::batch_read_descriptor(
+std::vector<std::pair<DescriptorItem, py::object>> PythonVersionStore::batch_read_descriptor(
         const std::vector<StreamId>& stream_ids,
         const std::vector<VersionQuery>& version_queries){
     
     auto vector_descriptors = batch_read_descriptor_internal(stream_ids, version_queries);
-    std::vector<std::pair<VersionedItem, py::object>> output_vector;
-    for (auto [version, metadata_proto] : vector_descriptors) {
+    std::vector<std::pair<DescriptorItem, py::object>> output_vector;
+    for (auto [descriptor, metadata_proto] : vector_descriptors) {
```

Comment:
Can use `auto&&` here

Reply:
Also, we use the term descriptor a lot for other things, please rename to `descriptor_item`